crates/hyperdrive-math/src/lp.rs

```diff
@@ -757,7 +752,7 @@ mod tests {
             alice
                 .advance_time(
                     rate,
-                    FixedPoint::from(config.checkpoint_duration) * fixed!(0.5e18),
+                    FixedPoint::from(config.checkpoint_duration).mul_down(fixed!(0.5e18)),
```

Comment:
why this switch from operators?

Reply:
i just did it to ensure it matched solidity